src/app/map/[id]/Tile/hierarchy-tile.static.tsx

```diff
@@ -0,0 +1,106 @@
+"use client";
```

Comment:
The only reason it needs to be a client component is because the use of useSearchParams and onClick on the button. Refactor this to use a prop URLInfo (probably containing searchParams and pathname if I had to guess), and a  next Link instead of the button + handler. Drill down the URLInfo prop accordingly (I think from the page.tsx file)